### DISCUSSION TITLE
Problem with directory structure

### DISCUSSION THREAD

**Discussion by jafi0101** [beetbox/beets#4621] [Q&A] [answered]
From my config.yaml
`paths:
    default: $albumartist/$album%aunique{}/$track - $title
    singleton: Non-Album/$artist - $title
    comp: Compilations/$album%aunique{}/$track - $title
    albumtype_soundtrack: Soundtracks/$album/$track $title`

I would like to use structure:
Artist - Album - Song

But for some reason beets change my structure to:
Artist - Artist - Album - Song

Why?

**Comment by JOJ0** [comment, marked as answer]
The config part you posted is quite difficult to read. Probably some line breaks got lost during copy/past. Would help if you fix that. Thanks a lot!

Educated guess around finding a solution to your problem: Double check your `paths` section. Check if yaml syntax is correct (indentation! spaces vs. tabs and so on)

A default like this should do the trick:

```
paths:
    default: Your_Albums_main_directory/$albumartist - $album%aunique{} - $title 
```

Do something similar with your comp: and singleton: lines!

Use pretend mode before really moving files (can take a long time, depending on you library size):

```
beet mv -p
```
